kvbc/src/categorization/blocks.cpp

```diff
@@ -108,6 +108,7 @@ ImmutableInput RawBlock::getUpdates(const std::string& category_id,
   ConcordAssert(categorires != nullptr);
   ConcordAssert(categorires->count(category_id) == 1);
   ImmutableInput data;
+  data.calculate_root_hash = update_info.tag_root_hashes.has_value();
```

Comment:
Nit: Not related to this PR, but we might change the names of these functions from updates to something related to input/output. Just came to mind, we can totally think about it later.